app/controllers/api/v1/medical_shifts_controller.rb

```diff
@@ -45,6 +45,18 @@ def update
         end
       end
 
+      def destroy
+        authorize(medical_shift)
+
+        result = MedicalShifts::Destroy.result(id: medical_shift.id.to_s)
+
+        if result.success?
+          render json: result.medical_shift, status: :ok
+        else
+          render json: result.error, status: :unprocessable_entity
```

Comment:
I think this kind of thing should be handled by api_controller, but I don't remember how is the flow

Reply:
https://github.com/espoo-dev/med_system_backend/issues/207 here we will treat it